esp_lcd_qemu_rgb/include/esp_lcd_qemu_rgb_struct.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Organization of the RGB QEMU panel registers */
+typedef struct rgb_qemu_dev_s {
+    union {
```

Comment:
should we add "volatile"?

Reply:
I added the `volatile` to this `typedef` and removed it from the definition